robot/web/web_c.go

```diff
@@ -30,14 +28,6 @@ import (
 	rutils "go.viam.com/rdk/utils"
 )
 
-var (
```

Comment:
This changes `web_c.go` back to the state it was in prior to https://github.com/viamrobotics/rdk/pull/3723

Determining whether `rtp_passthrough` functionality or the gostream `GetImage` code path should be used is now handled by stream/server.go and stream/state/state.go